packages/Python/lldbsuite/test/lang/swift/member_lookup_crash/main.swift

```diff
@@ -10,7 +10,8 @@ extension Measurement where UnitType == UnitAngle {
   }
 
   func f() {
```

Comment:
Can you rename this test to something more meaningful than "crash"?

Reply:
Sure, the test contained the word "crash" to begin with but I agree it can be renamed to something more useful :)